packages/cli-repl/test/e2e.spec.ts

```diff
@@ -296,6 +297,14 @@ describe('e2e', function() {
       await result;
       shell.assertContainsError('interrupted');
     });
+    it('behaves normally after an exception', async() => {
+      await shell.executeLine('throw new Error()');
+      await new Promise((resolve) => setTimeout(resolve, 100));
+      shell.kill('SIGINT');
+      await shell.waitForPrompt();
+      await new Promise((resolve) => setTimeout(resolve, 100));
+      assert(!/interrupted/.test(shell.output));
```

Comment:
Would it be good to have a `shell.assertNotContainsOutput` for consistency, that also outputs the diff?

Reply:
Sure, makes sense